src/components/root/MainSection.tsx

```diff
@@ -0,0 +1,110 @@
+import { getModuleEmoji } from '@src/features/contentEnhancers/emoji/modules';
+import { RootContent } from '@src/features/parser';
+import { ModuleContent } from '@src/features/parser/root/parseRootPageContent';
+import { QuickLinkLink } from '@src/features/quickLinks/components/quickLink/QuickLinkLink';
+import React from 'react';
+
+interface MainSectionProps {
+  content: RootContent;
+}
+
+const MainSection: React.FC<MainSectionProps> = ({ content }) => {
```

Comment:
just style issue:

change this to just `export function ...`, 

(again, no docs)